src/context.h

```diff
@@ -161,8 +161,6 @@ struct ly_ctx;
  *
  * Options to change context behavior.
  *
- * Note that the flags 0xFF00 are reserved for internal use.
- *
```

Comment:
Could have been in a separate commit but I take it that you checked this and found that there are no internal flags anymore.

Reply:
Yes, I did not find any internal flags. I will make it a special commit, which will be the first in this PR.